pyodide-build/pyodide_build/pywasmcross.py

```diff
@@ -698,7 +605,15 @@ def handle_command(
 
 
 def compiler_main():
-    build_args = BuildArgs(**PYWASMCROSS_ARGS)
+    build_args = CrossCompileArgs(
+        pkgname=PYWASMCROSS_ARGS.pop("pkgname"),
+        cflags=PYWASMCROSS_ARGS.pop("cflags"),
+        cxxflags=PYWASMCROSS_ARGS.pop("cxxflags"),
+        ldflags=PYWASMCROSS_ARGS.pop("ldflags"),
+        target_install_dir=PYWASMCROSS_ARGS.pop("target_install_dir"),
+        pythoninclude=PYWASMCROSS_ARGS.pop("pythoninclude"),
+        exports=PYWASMCROSS_ARGS.pop("exports"),
+    )
```

Comment:
Why do you use `pop` here?

Reply:
No special reason. I just saw that `pop` is used elsewhere in pywasmcross, but yeah just accessing the keys will be fine.